### DISCUSSION TITLE
¡Un juego de código abierto sobre el aprendizaje de Git!

### DISCUSSION THREAD

**Discussion by JorgeSoberanis** [LaunchX-InnovaccionVirtual/CursoIntroPython#310] [Show and tell]
Buenas a todos, les comento de este increíble recurso, es un video juego donde se expresan todas las funciones de GIT y como estas interactúan con los documentos, desde el clásico git init hasta los git command --help 
 
Y lo mejor de todo es que es completamente gratis! 💯 

https://ohmygit.org/

**Comment by ErikGustavoAvendanoG** [comment]
Gracias compañero
